automatic/1password/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName = '1Password'
+
+[array]$key = Get-UninstallRegistryKey -SoftwareName '1Password*'
+
+if ($key.Count -eq 1) {
+  $key | ForEach-Object {
+    $packageArgs = @{
+      packageName    = $packageName
+      fileType       = 'EXE'
+      silentArgs     = if ($env:ChocolateyPackageName -eq "1password4") { '/SILENT' } else { 'uninstall' }
```

Comment:
> Appveyor still has some issue with mixing up 1password and 1password4, hence the compatibility line here.

It is not that it gets mixed up, but rather they are part of AU streams. The script that is responsible for running the tests only picks up the first stream (it actually don't have knowledge about streams), and since it is the 1password stream that is the first that one is picked up.